### ISSUE TITLE
Jack o lanterns are not possible to craft.

### ISSUE THREAD

**Issue by DaddySpin** [LudoCrypt/CarveMyPumpkin#3]
After carving a pumpkin and combining it with a torch it gives me a jack'o'lantern with no nbt tags on it and when i place it and then break it the item is not dropped.

**Comment by LudoCrypt** [comment]
Are you using REI, if so, update it.

**Comment by DaddySpin** [comment]
I'm using the latest of both REI and Carve my Pumpkin and I still am having the problem.

Carve a pumpkin with a carver and then craft it with a torch and then I receive the item carvepump:jack_o_lantern it does not have a face on it and no nbts tags on it and when i place and break it, the item does not drop.

I did not use REI in the process of crafting.

**Comment by duncy** [comment]
I also had this exact issue, while on the most current versions of each mod.

EDIT: I am also using KubeJS, I have tried removing it on my client and it works on singleplayer now. However if it is installed on a server, the issue persists regardless of whether it is installed on the client or not. Before this I tried disabling REI and it had no effect, so I think this is caused by KubeJS.

**Comment by JuergenSchT** [comment]
I'm also having this issue, too.
Tested FabricAPI , REI 5.6.2 and CMP 1.0.3 on singleplayer and multiplayer. The presence or absence of REI on the client, server or both has no effect on the bug,
It's not just jack-o'-lanterns without/with incorrect NBT that will not drop, it also happens with carved pumpkins/melons without/with incorrect NBT. IMO this is not a bug.
Any jack-o'-lantern/melon with the proper NBT can be broken and placed with no problems at all, it's the newly-crafted ones that are not receiving their NBT.
I also tested CMP 1.0.2, and the bug of jack-o'-lanterns/melons not receiving their NBT is also present there.

**EDIT**: After further testing, the bug doesn't happen with KubeJS uninstalled [on the client in singleplayer, or installed only on the server in multiplayer]. It happens even if there is no KubeJS script loaded that modifies recipes.
